packages/webapp/src/utils/constants.js

```diff
@@ -128,6 +136,7 @@ const NEED_OWNERSHIP_PROOF = 'land/need-ownership-proof'
 const NEED_METRIC = 'land/need-metric'
 const NEED_MANAGEMENT_PLAN = 'land/need-management-plan'
 const NEED_LEGAL_AGREEMENT = 'land/need-legal-agreement'
+const NEED_LOCAL_CHARGE = 'land/need-local-charge'
```

Comment:
Please replace with **const NEED_LOCAL_LAND_CHARGE = 'land/need-local-land-charge'** and update dependencies